tests/test_variants.py

```diff
@@ -281,6 +281,7 @@ def test_ensure_valid_spec_on_run_and_test(testing_workdir, testing_config, capl
     assert "Adding .* to spec 'pytest-mock  1.6'" not in text
 
 
+@pytest.mark.skipif(sys.platform == "darwin", reason="Unsatisfiable dependencies for M1 MacOS: {'bzip2=1.0.6'}")
```

Comment:
We don't want to skip these because they still work on Intel macOS, either explicitly test for M1 machines or mark them as `xfail`:

```suggestion
@pytest.mark.skipif(on_mac and platform.machine() == "arm64", reason="Unsatisfiable dependencies for M1 MacOS: {'bzip2=1.0.6'}")
```